README.md

```
@@ -2,80 +1,233 @@
+## BackEnd
+
+![백엔드](packages/shared-assets/readme/back-end.png)
+
+## FrontEnd
+
+![프론트엔드](packages/shared-assets/readme/front-end.png)
+
+## ETC
+
+<div  align="center">
+<img src="https://img.shields.io/badge/pnpm-F69220?style=for-the-badge&logo=pnpm&logoColor=white"> <img src="https://img.shields.io/badge/Prettier-F7B93E?style=for-the-badge&logo=Prettier&logoColor=white"> <img src="https://img.shields.io/badge/ESlint-4B32C3?style=for-the-badge&logo=ESlint&logoColor=white"> 
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)